File: src/3162.find-the-number-of-good-pairs-i.3446/solution.ts
function numberOfPairs (nums1: number[], nums2: number[], k: number): number {
    let cnt = 0;
    for (let i = 0; i < nums1.length; i++) {
        for (let j = 0; j < nums2.length; j++) {
            if (nums1[i] % (nums2[j] * k) === 0) {
                cnt++;
            }
        }
    }
    return cnt;
}

File: src/3162.find-the-number-of-good-pairs-i.3446/solution.java
class Solution {
    public int numberOfPairs(int[] nums1, int[] nums2, int k) {
        int result = 0;
        for(int n1 : nums1){
            for(int n2 : nums2){
                if(n1%(n2*k) == 0){
                    result++;
                }
            }
        }
        return result;
    }
}